### PR TITLE
xRaddextexp - replaced yaml files with header table

### PR DESCRIPTION
**Pull request by nitheshsrikanth-mcw**

### CODE REVIEW

**Review by dsharlet — COMMENTED** [review]
Same comments as #7408

File: tools/generate-raddextexp-test.py
Comment (dsharletg) [inline tools/generate-raddextexp-test.py:1]: Delete this file

**Review by dsharlet — APPROVED** [review]

**Comment by dsharlet** [comment]
Update kernel declarations?

File: test/f32-raddextexp.cc [hunk 1]
```diff
@@ -2,902 +2,112 @@
...
-    TEST_REQUIRES_X86_AVX2;
-    for (size_t elements = 1; elements < 80; elements++) {
-      RAddExtExpMicrokernelTester()
-        .elements(elements)
-        .Test(xnn_f32_raddextexp_ukernel__avx2_p5_u80_acc2);
-    }
+#define XNN_TEST_RADDEXTEXP_ELEMENT_DIV(ukernel, arch_flags, element_tile, ...)                                        \
+  TEST(ukernel, element_gt)                                                                                            \
+  {                                                                                                                    \
+    for (size_t element_size = element_tile * 2; element_size < element_tile * 10; element_size += element_tile) {     \
```
Comment (dsharletg) [inline test/f32-raddextexp.cc:87]: `TEST_REQUIRES_ARCH_FLAGS(arch_flags)`?

Comment (nitheshsrikanth-mcw) [reply test/f32-raddextexp.cc:89]: Thanks for pointing out the missing TEST_REQUIRES_ARCH_FLAGS(arch_flags). The issue with the illegal instruction for AVX-512 has now been resolved and now the tests are passing.

**Review by dsharlet — APPROVED** [review]

**Review by dsharlet — APPROVED** [review]

**Pull request merged**